### ISSUE TITLE
Example Prompts

### ISSUE THREAD

**Issue by steven850** [logtd/ComfyUI-MochiEdit#18]
Could you share the prompts used for the example videos? along with settings, Specifically for the edit of the man on the couch. 
im trying to just change the color of a shirt in a video, and my output is nothing like the original video. So not sure if its a setting, or if the prompt is just to simple.

**Comment by tuckerdarby** [comment]
I don't have those any more but they were something like
" a man wearing a white collared shirt and a top hat", "an elderly woman wearing a white collared shirt", "a man wearing a white collared shirt, cgi style", etc.

It took some tuning to find settings that worked well with them between the eta, start step and end step.

**Comment by nosy-b** [comment]
Is there a way to input an image as reference rather than write text prompts?
Thx!

**Comment by logtd** [comment]
No, the current version of Mochi doesn't have that capability -- hopefully they'll release an I2V model one day!
